src/modules/paymentProcessor/PP_Streaming_v1.sol

```diff
@@ -140,9 +145,16 @@ contract PP_Streaming_v1 is Module_v1, IPP_Streaming_v1 {
     function init(
         IOrchestrator_v1 orchestrator_,
         Metadata memory metadata,
-        bytes memory /*configData*/
+        bytes memory configData
     ) external override(Module_v1) initializer {
         __Module_init(orchestrator_, metadata);
+
+        (uint _defaultStart, uint _defaultCliff, uint _defaultEnd) =
+            abi.decode(configData, (uint, uint, uint));
+
+        defaultStart = _defaultStart;
```

Comment:
Maybe create some setters for this, so it can be modified later? Also, we can check for _validTimes()

Reply:
Who should be able to call these setters?